buildsystem/doxygen.cmake

```diff
@@ -39,6 +39,8 @@ function(doxygen_configure)
 
 		# adapt doxygen config
 		configure_file("${BUILDSYSTEM_DIR}/templates/Doxyfile.in" "${CMAKE_BINARY_DIR}/Doxyfile" @ONLY)
+		configure_file("${BUILDSYSTEM_DIR}/templates/doxygen_custom.css.in"
```

Comment:
do you have to change values in the css at configure-time? If yes, this is the way to go  (even though you should output the file to `"${CMAKE_BINARY_DIR}/doxygen_custom.css"`). If no, you can add the css path in the `Doxyfile.in` (which is configured above and can also replace variables in that process so the css is found in the source dir).